packages/specification/eps-prescription-status-update-api.yaml

```diff
@@ -53,15 +55,37 @@ info:
     For more details see [Network access for APIs](https://digital.nhs.uk/developer/guides-and-documentation/network-access-for-apis).
 
     ## Security and Authorisation
-    This API only supports application-restricted access, meaning we authenticate the calling application but not the end user.
+    This API only supports [application-restricted access](https://digital.nhs.uk/developer/guides-and-documentation/security-and-authorisation#application-restricted-apis), meaning we authenticate the calling application but not the end user.
 
+    ### Application-restricted Access
     The end user could be:
-    a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
-    not present at all - for example as part of a back end process integrated with a shipping system
+
+    * a healthcare worker - in which case you must ensure they are authenticated and suitably authorised locally
+    * a patient - in which case you must ensure they are authenticated locally
```

Comment:
It wouldn't be used by a patient, we excluded that intentionally

Reply:
0e9845dc631bf9cd656b409d4616911c2b3f874d